dashboard/test/ui/runner.rb

```diff
@@ -23,14 +23,16 @@
 require 'colorize'
 require 'open3'
 require 'parallel'
+require 'socket'
 
 require 'active_support/core_ext/object/blank'
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+GIT_BRANCH = GitUtils.current_branch
 COMMIT_HASH = RakeUtils.git_revision
 S3_LOGS_BUCKET = 'cucumber-logs'
-S3_LOGS_PREFIX = GitUtils.current_branch
+S3_LOGS_PREFIX = "#{Socket.gethostname}/#{GIT_BRANCH}"
```

Comment:
Since we've generalized usage of the S3 prefix to allow it to contain slashes, we could even do:
```
ENV['CI'] ? "circle/#{ENV['CIRCLE_BUILD_NUM']" : Socket.gethostname
```

Does this even matter, though?  We don't currently generate a status page for Circle runs, although I guess we could...